app/javascript/components/carbon-charts/stackBarChart.js

```diff
@@ -24,7 +24,7 @@ const StackBarChartGraph = ({ data, title }) => {
   };
 
   return (
-    <StackedBarChart data={data} options={options} />
+    <StackedBarChart data={data} options={chart_options? chart_options : options} />
```

Comment:
Style (I'm surprised the bot didn't see this)

```suggestion
    <StackedBarChart data={data} options={chart_options ? chart_options : options} />
```